@xen-orchestra/proxy/src/app/_disposeResourceOnStop.js

```diff
@@ -0,0 +1,18 @@
+import Resource from 'promise-toolbox/_Resource'
+
+export function disposeResourceOnStop(res, [getHooks], args) {
```

Comment:
Why `getHooks` is in an array?

Reply:
Because it is the syntax that i choose for the decorator `decorateResult` to deal with the transformers args and the decorated method args.